storage_service/locations/models/async.py

```diff
@@ -2,6 +2,7 @@
 # stdlib, alphabetical
 import cPickle
```

Comment:
In Py3 `cPickle` is just `pickle` right, so if we're going to anticipate supporting Python 3 here, should we import `pickle` as `cPickle`?